src/components/AddBook.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+// import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { addBookAction } from '../redux/books/books';
+
+const AddBook = () => {
+  // const { bookItem } = props;
+  const dispatch = useDispatch();
+
+  const clickHandler = (e) => {
+    e.preventDefault();
+    const id = uuidv4();
+    const title = e.target[0].value;
+    const author = e.target[1].value;
+
+    if (title && author) {
+      dispatch(addBookAction(id, title, author));
+      e.target.title.value = '';
+      e.target.title.value = '';
```

Comment:
- [ ] It will be better if you change this code with these because it raises an error in the console:
```js
      e.target[0].value = '';
      e.target[1].value = '';
```